Makefile

```diff
@@ -8,7 +8,7 @@ OPERATOR_IMAGE     ?= kubevirt/hyperconverged-cluster-operator
 REGISTRY_NAMESPACE ?=
 
 build: $(SOURCES) ## Build binary from source
-	go build -i -ldflags="-s -w" -o _out/hyperconverged-cluster-operator ./cmd/manager > /dev/null
+	go build -i -ldflags="-s -w" -o _out/hyperconverged-cluster-operator ./cmd/manager
```

Comment:
ups, didn't want to commit this. But, why do you do this?

Reply:
probably shouldn't be there since it's suppressing stdout.

Reply:
I had this eerie suspicion that this was something I had done...but I have no answer to your question why this was done.

Reply:
ack, then I will leave it :+1: